packages/ingest/test/test-consumer.js

```diff
@@ -10,11 +10,13 @@ const { Consumer } = require('../consumer');
 
 const timeToReceiveMessages = 200; // ms
 const timeLimitModifier = 50;
+const sqsMessage = { Body: 'message', MessageId: 'id' };
 let testConsumer;
 
 async function stubReceiveSQSMessages(_url, { numOfMessages }) {
   await delay(timeToReceiveMessages);
-  return Array.apply(undefined, { length: numOfMessages }).map(() => 'i am a message'); // eslint-disable-line prefer-spread
+  // eslint-disable-next-line prefer-spread
+  return Array.apply(undefined, { length: numOfMessages }).map(() => sqsMessage);
```

Comment:
No objection here, but why the change?

Reply:
It made more sense to me because it specifies disabling that rule for the next line `eslint-disable-next-line`, but I am not sure it's a good enough reason to keep the change